src/components/SubCategory.js

```diff
@@ -24,36 +23,40 @@ const StyledSubCategory = styled.div`
     flex-wrap: wrap;
     margin-left: 10px;
   }
-`;
+`
 
 const subCategoryInfo = {
-    id: 'unique',
-    title: 'Books/Movies/Music',
-  }
-      
-const subCategoryEntries = [
-  {title: "Mistborn", amount: 9.99},
-  {title: "Name of the Wind", amount: 24.95},
-  {title: "The Dark Knight", amount: 8.99},
-  {title: "The Marvelous Mrs. Maisel", amount: 4.92},
-  {title: "Mumford & Sons Vinyl", amount: 15.65}
-];
+  id: 'unique',
```

Comment:
this was meant to be a placeholder, the point was that the `id` should be a unique title